src/firebase/api.ts

```diff
@@ -10,12 +10,12 @@ const firebaseConfig = {
   messagingSenderId: '1047480032559',
   appId: '1:1047480032559:web:dc4c9d845104ba1f8c55da',
   measurementId: 'G-6WS3MMV659',
-};
+}
 
-firebase.initializeApp(firebaseConfig);
+firebase.initializeApp(firebaseConfig)
 
-const emailsIDCollection = firebase.firestore().collection('emails_id');
+const emailsIDCollection = firebase.firestore().collection('emails_id')
 
-const addEmail = (email: string): Promise<void> => emailsIDCollection.doc(email).set({ email });
+export const addEmail = (email: string): Promise<void> => emailsIDCollection.doc(email).set({ email })
 
-export default addEmail;
+export default firebase
```

Comment:
why are you exporting `firebase` here? Usually you just want the function

Reply:
Because the functions were being created in componentDidMount, simply for readability during development